### DISCUSSION TITLE
Update https://github.com/awslabs/aws-c-common submodule

### DISCUSSION THREAD

**Discussion by theoparis** [aws/aws-sdk-cpp#2837] [General]
Without the latest commit (https://github.com/awslabs/aws-c-common/commit/901405ec8c54da3a773e9699de1cbb43730267f3), aws-sdk-cpp fails to build with the following error:
```
CMake Error at build/_deps/aws-sdk-cpp-src/crt/aws-c-common/CMakeLists.txt:228 (simd_add_definitions):
  Unknown CMake command "simd_add_definitions".
```

Not sure what's happening but updating the submodule with the latest aws-c-common commit seems to fix it 🤔

**Comment by jmklix** [comment]
I'm not sure what might have caused this, but I'm glad you got it working. Please let us know if you see this happen again with the latest releases of the sdk and sub-modules.
